update.sh

```diff
@@ -72,6 +72,11 @@ for version in "${versions[@]}"; do
 			s/libssl-dev/libssl1.0-dev/;
 		'
 	fi
+	if [[ "$version" = 1.* ]]; then
+		sedExpr+='
+			s/linux-glibc/linux2628/;
```

Comment:
`linux2628` was renamed to `linux-glibc`: https://github.com/haproxy/haproxy/commit/61cf0dc9a00944454d9635cd3675fd50e6fb5822

musl supports everything that is enabled by the `glibc` target, so this is fine for alpine as well (biggest issue apparently being `USE_GETADDRINFO` which we already enabled manually).